collector/Makefile

```diff
@@ -48,12 +48,12 @@ package: build
 
 .PHONY: publish
 publish:
-	aws lambda publish-layer-version --layer-name $(LAYER_NAME) --zip-file fileb://$(BUILD_SPACE)/opentelemetry-collector-layer-$(GOARCH).zip --compatible-runtimes nodejs14.x nodejs16.x nodejs18.x java11 python3.8 python3.9 python3.10 --query 'LayerVersionArn' --output text
+	aws lambda publish-layer-version --layer-name $(LAYER_NAME) --zip-file fileb://$(BUILD_SPACE)/opentelemetry-collector-layer-$(GOARCH).zip --compatible-runtimes nodejs14.x nodejs16.x nodejs18.x java11 python3.8 python3.9 python3.10 3.11 --query 'LayerVersionArn' --output text
```

Comment:
typo in the name of the runtime? `3.11` -> `python3.11`?

[reference](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html)

Reply:
Yeah, typo :) thanks!